examples/tutorials/curation/plot_1_automated_curation.py

```diff
@@ -0,0 +1,248 @@
+balanced_accuracy = balanced_accuracy_score(human_labels, predictions)
+
+plt.imshow(conf_matrix)
+for (index, value) in np.ndenumerate(conf_matrix):
+    plt.annotate( str(value), xy=index, color="white", fontsize="15")
+plt.xlabel('Predicted Label')
+plt.ylabel('Human Label')
+plt.xticks(ticks = [0, 1], labels = list(label_conversion.values()))
+plt.yticks(ticks = [0, 1], labels = list(label_conversion.values()))
+plt.title('Predicted vs Human Label')
+plt.suptitle(f"Balanced Accuracy: {balanced_accuracy}")
+plt.show()
+
+
+##############################################################################
+# Here, there are several false positives (if we consider the human labels to be "the truth").
+#
+# Next, we can also see how the model's confidence relates to the probability that the model
+# label matches the human label
+#
+# This could be used to set a threshold above which you accept the model's classification,
+# and only manually curate those which it is less sure of.
+
+
+def calculate_moving_avg(label_df, confidence_label, window_size):
+
+    label_df[f'{confidence_label}_decile'] = pd.cut(label_df[confidence_label], 10, labels=False, duplicates='drop')
+    # Group by decile and calculate the proportion of correct labels (agreement)
+    p_label_grouped = label_df.groupby(f'{confidence_label}_decile')['model_x_human_agreement'].mean()
+    # Convert decile to range 0-1
+    p_label_grouped.index = p_label_grouped.index / 10
+    # Sort the DataFrame by confidence scores
+    label_df_sorted = label_df.sort_values(by=confidence_label)
+
+    p_label_moving_avg = label_df_sorted['model_x_human_agreement'].rolling(window=window_size).mean()
+
+    return label_df_sorted[confidence_label], p_label_moving_avg
+
+confidences = labels['probability']
+
+# Make dataframe of human label, model label, and confidence
+label_df = pd.DataFrame(data = {
+    'human_label': human_labels,
+    'decoder_label': predictions,
+    'confidence': confidences},
+    index = sorting_analyzer.sorting.get_unit_ids())
+
+# Calculate the proportion of agreed labels by confidence decile
+label_df['model_x_human_agreement'] = label_df['human_label'] == label_df['decoder_label']
+
+p_agreement_sorted, p_agreement_moving_avg = calculate_moving_avg(label_df, 'confidence', 3)
+
+# Plot the moving average of agreement
+plt.figure(figsize=(6, 6))
+plt.plot(p_agreement_sorted, p_agreement_moving_avg, label = 'Moving Average')
+plt.axhline(y=1/len(np.unique(predictions)), color='black', linestyle='--', label='Chance')
+plt.xlabel('Confidence'); #plt.xlim(0.5, 1)
+plt.ylabel('Proportion Agreement with Human Label'); plt.ylim(0, 1)
+plt.title('Agreement vs Confidence (Moving Average)')
+plt.legend(); plt.grid(True); plt.show()
+
+##############################################################################
+# In this case, you might decide to only trust labels which had confidence over above 0.86.
+#
+# A more realistic example: Neuropixels data
```

Comment:
I found this section quote confusing, I would suggest to rename it like 'Using other models' or something, and move it about the assessment vs. human labels section. Then the tutorial would read like:

1) introduction
2) how to run on tetrode model
3) explain what models are available, example of using a different type of model
4) comparing outputs vs human curated data

Reply:
I like having the model performance after the toy_model stuff, because it uses the same dataset. I think we should just link to this section above, and try and improve it a bit

Reply:
The model we're using here is being updated. So I'll do these updates once that's finished.

Reply:
Model is updated, so this section is now significantly updated.